pkg/deploy/generator/resources_rp.go

```diff
@@ -334,69 +308,6 @@ func (g *generator) rpLB() *arm.Resource {
 	}
 }
 
-func (g *generator) rpLBInternal() *arm.Resource {
-	return &arm.Resource{
-		Resource: &mgmtnetwork.LoadBalancer{
-			Sku: &mgmtnetwork.LoadBalancerSku{
-				Name: mgmtnetwork.LoadBalancerSkuNameStandard,
-			},
-			LoadBalancerPropertiesFormat: &mgmtnetwork.LoadBalancerPropertiesFormat{
-				FrontendIPConfigurations: &[]mgmtnetwork.FrontendIPConfiguration{
-					{
-						FrontendIPConfigurationPropertiesFormat: &mgmtnetwork.FrontendIPConfigurationPropertiesFormat{
-							Subnet: &mgmtnetwork.Subnet{
-								ID: to.StringPtr("[resourceId('Microsoft.Network/virtualNetworks/subnets', 'rp-vnet', 'rp-subnet')]"),
-							},
-						},
-						Name:  to.StringPtr("dbtoken-frontend"),
-						Zones: &[]string{},
-					},
-				},
-				BackendAddressPools: &[]mgmtnetwork.BackendAddressPool{
-					{
-						Name: to.StringPtr("rp-backend"),
-					},
-				},
-				LoadBalancingRules: &[]mgmtnetwork.LoadBalancingRule{
-					{
-						LoadBalancingRulePropertiesFormat: &mgmtnetwork.LoadBalancingRulePropertiesFormat{
-							FrontendIPConfiguration: &mgmtnetwork.SubResource{
-								ID: to.StringPtr("[resourceId('Microsoft.Network/loadBalancers/frontendIPConfigurations', 'rp-lb-internal', 'dbtoken-frontend')]"),
-							},
-							BackendAddressPool: &mgmtnetwork.SubResource{
-								ID: to.StringPtr("[resourceId('Microsoft.Network/loadBalancers/backendAddressPools', 'rp-lb-internal', 'rp-backend')]"),
-							},
```

Comment:
This is the only removal that concerns me here and generally with the LB, it could just be an odd naming choice, but to be sure, this isn't removing the LB config for anything related to the RP Backend service, right?

Reply:
`rp-lb-internal` is currently being used for gateway to request `dbtoken` service in `RP`, apart from that in my knowledge it is not used for anything else. Now since dbtoken service is being removed, there is no use of this LB